### DISCUSSION TITLE
[RFC/Translation] Use Transifex for helping localise the application?

### DISCUSSION THREAD

**Discussion by adityaruplaha** [Heroic-Games-Launcher/HeroicGamesLauncher#221] [Ideas]
Currently all translations need to be updated using PRs. While this is fine, maybe consider using a service like Transifex for getting more people helping with this? Not everyone is comfortable dealing with large JSON files.

Documentation: https://docs.transifex.com/javascript-sdk/localize-react-applications

This is going to need a application rework though I think, which is going to be easier now than later as the code volume is lesser. Just a suggestion though, please give your opinion on this.

**Comment by flavioislima** [comment]
I never heard about this tool to be honest, But as far as I could look, I could not find a NodeJS integration. And we have some messages on the electron backend that needs translations.
But you're right, the way we are doing the translations now are not the ideal. We need to think about something else.

**Comment by adityaruplaha** [comment]
We've decided on using Weblate for now, so can this be closed?

**Reply by flavioislima** [reply]
yep!

**Reply by adityaruplaha** [reply]
Kindly close it then, I don't have the perms.
